src/styles/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  font-family: $roboto-font;
+  color: $text-color;
+  font-weight: 400;
+  transition: all 300ms linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
The transform property in hover state increases the size of the card, which might cause layout shifts. Make sure this does not affect the surrounding elements and the overall layout.